src/sql/platform/connection/common/constants.ts

```diff
@@ -34,3 +34,7 @@ export const passwordChars = '***************';
 export const sqlLogin = 'SqlLogin';
 export const integrated = 'Integrated';
 export const azureMFA = 'AzureMFA';
+
+/* CMS constants */
+export const cmsProviderName = 'MSSQL-CMS';
+export const cmsProviderDisplayName = 'Microsoft SQL Server (CMS)';
```

Comment:
This should be localized if it's a visible display name

Reply:
Does SQL Sever need to be localized? We haven't localized that anywhere

Reply:
Not likely - but what about CMS? That isn't a product name so I'd imagine it should be localized. And regardless any display strings should really always be marked as localizable even if they only contain strings such as SQL Server in case there's a case we don't know about (or is changed) that requires displaying those names differently in different languages.

Reply:
The provider name is the same for all languages I believe. I'm not sure how `CMS` would be localized itself as a string.

Reply:
This is for the displayName only - yes the cmsProviderName should always be a constant value. But displayName implies that it's being displayed in the UI somewhere.

I don't know offhand whether CMS the acronym should be localized but Central Management Server definitely is - e.g. https://docs.microsoft.com/fr-fr/sql/relational-databases/administer-multiple-servers-using-central-management-servers?view=sql-server-2017 so it would make sense to me that CMS would also be different.